content/topics/logging.md

```diff
@@ -17,7 +17,6 @@ packages = [
   "syslog",
   "flexi_logger",
   "sentry",
-  "slog",
```

Comment:
slog seems reasonably active at first glance and seems to have decent download numbers

Reply:
Yup it's mostly because of [their notice on their readme](https://github.com/slog-rs/slog/blob/master/README.md) that I removed it, but I don't mind leaving it if you want.

```
You might consider using tracing instead

It's been a while since slog was created and it served Rust community well all this time. It remains a stable, featureful and battle-tested library, used in many important projects.

In last few years, another ecosystem for Rust was created with similar features and a very good support for debugging async code and already larger dev team and community.

Please check [tracing](https://github.com/tokio-rs/tracing) and see if it is more suitable for your use-case. It seems that it is already a go-to logging/tracing solution for Rust.
```

Reply:
ah, I see, thanks :)